xc/xc7/tests/soc/litex/fixup_xdc.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+"""
+Currently, litex outputs XDC constraints in which the create_clock commands
+cannot be correctly parsed yet by the XDC yosys plugin.
+
+Example of failing XDC command:
+    create_clock -name clk100 -period 10.0 [get_nets clk100]
```

Comment:
I thought inspection commands had landed?  @tmichalak ?

Reply:
Not yet, the PR is [here](https://github.com/SymbiFlow/yosys-symbiflow-plugins/pull/42).
 Still needs some work to address review comments.